src/container/include/truststore-setup.sh

```diff
@@ -2,16 +2,16 @@
 
 set -e
 
-function genpass() {
-    echo "$(< /dev/urandom tr -dc _A-Z-a-z-0-9 | head -c32)"
+genpass() {
+    printf '%s' "$(< /dev/urandom tr -dc _A-Z-a-z-0-9 | head -c32)"
```

Comment:
I guess this whole line can probably be replaced with just `< /dev/urandom tr -dc _A-Z-a-z-0-9 | head -c32` and strip out the rest. It should naturally print to stdout this way, no need to wrap it into a string and format that to print with printf/echo.